haystack/nodes/file_converter/csv.py

```diff
@@ -29,11 +29,23 @@ def convert(
         id_hash_keys: Optional[List[str]] = None,
     ) -> List[Document]:
         """
-        Load CVS file and convert it to documents.
-
-        :param file_path: Path to a CSV file containing two columns.
-            The first will be interpreted as a question, the second as content.
-        :returns: List of document, 1 document per line in the CSV.
+                Load a CVS file containing question-answer pairs and convert it to Documents.
+
+                :param file_path: Path to the CSV file you want to convert. The file must have two columns called 'question' and 'answer'.
+                    The first will be interpreted as a question, the second as content.
+                :param meta: A dictionary of metadata key-value pairs that you want to append to the returned document. It's optional.
+                :param remove_numeric_tables: This option uses heuristics to remove numeric rows from the tables.
+                    The tabular structures in documents might be noise for the Reader model if it
+                    doesn'tt have table parsing capability for finding answers. Tables
+                    may also have long strings that could be possible candidates for searching answers.
+                    So this option retains the rows containing strings. This parameter is optional. Possible values are: `True` and `False`.
+                :param valid_languages: Takes a list of languages in the [ISO 639-1](https://en.wikipedia.org/wiki/ISO_639-1) format and validates them. Use this option to check for encoding errors. If the extracted text is not one of the valid languages, it may mean there's an encoding error that results in garbled text. This parameter is optional.
```

Comment:
Same here as for `remove_numeric_tables`, `valid_languages` is not used by this node.